docker-app/qfieldcloud/core/serializers.py

```diff
@@ -523,3 +532,106 @@ class LatestPackageSerializer(serializers.Serializer):
     package_id = serializers.UUIDField()
     packaged_at = serializers.DateTimeField()
     data_last_updated_at = serializers.DateTimeField()
+
+
+class TeamSerializer(serializers.ModelSerializer):
+    organization = serializers.CharField(source='team_organization.username', read_only=True)
+    
+    class Meta:
+        model = Team
+        fields = ('username', 'organization')
+
+class TeamMemberSerializer(serializers.ModelSerializer):
+    member = serializers.StringRelatedField()  
+    
+    class Meta:
+        model = TeamMember
+        fields = ('team', 'member')
+
+class AddMemberSerializer(serializers.Serializer):
+    username = serializers.CharField(required=True)
+    email = serializers.EmailField(required=False, allow_blank=True)
+
+    def validate(self, data):
+        username = data.get('username')
+        user = self._validate_user_exists(username)
+        organization = self.context.get('organization')
+
+        self._validate_user_membership(user, organization)
+
+        data['user'] = user
+        data['organization'] = organization
+
+        return data
+
+    def _validate_user_exists(self, username):
+        if not username:
+            raise serializers.ValidationError("Username is required.")
+        
+        try:
+            user = User.objects.get(username=username)
+        except User.DoesNotExist:
+            raise serializers.ValidationError("User does not exist.")
+        
+        return user
+
+    def _validate_user_membership(self, user, organization):
+        is_org_member = OrganizationMember.objects.filter(
+            organization=organization,
+            member=user
+        ).exists()
+        
+        is_org_owner = organization.organization_owner == user
+        
+        if not (is_org_member or is_org_owner):
+            raise serializers.ValidationError("User is not a member of the organization.")
```

Comment:
You can remove these, they should not be validated in the serializer, but the view that calls the serializer. Check `core/views/members_views.py` for example.